src/renderer/features/operations/OperationsValidation/lib/validation/balance.ts

```diff
@@ -31,21 +31,17 @@ function insufficientBalanceForFee(
     isNative,
     isProxy,
     isMultisig,
-    isXcm,
   }: TransferFeeStore,
   config: Config = { withFormatAmount: true },
 ) {
-  if (isXcm && !isNative) {
-    if (isLteThanBalance(fee, balance)) {
-      return true;
-    }
+  if (!isNative) {
```

Comment:
Working with Polkadot asset hub involves Myth balance and Dot fee. So not only `xcm` cases must be validated with native balance.